submissions/JuliaNovak/tiny-js-world/index.js

```diff
@@ -0,0 +1,76 @@
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Inhabitant {
+   constructor(species, name, gender, legs, hands, saying) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.legs = legs;
+      this.hands = hands;
+      this.saying = saying;
+      this.properties = [
+         "name",
+         "species",
+         "gender",
+         "legs",
+         "hands",
+         "saying",
+      ];
+   }
+
+   get props() {
+      return this.properties.map((value) => this[value]);
+   }
+
+   display(inhabitantsArr) {
+      inhabitantsArr.map((inhabitant) => {
+         print(inhabitant.props);
+      });
+   }
+}
+
+class Human extends Inhabitant {
+   constructor(species, name, gender, saying) {
+      super(species, name, gender);
```

Comment:
What `super()` actually is and what it does?
The [docs on the original projetc repo](https://github.com/OleksiyRudenko/a-tiny-JS-world#leveling-your-skills-up) offers 3 articles specifically dedicated to `super`.
Once you find the answer you will fix your code.

Reply:
Thanks for the links!
The problem is that I've read these articles a couple of times but I still don't see the mistake. Maybe u could point to it or give some more hints?
Answering the question, I think that super here acts as a function and it references the variables in the parent class